addons/calendar/data/mail_data_various.xml

```diff
@@ -5,6 +5,14 @@
             <field name="category">meeting</field>
         </record>
 
+        <record id="mail_activity_data_scheduled_call" model="mail.activity.type">
+            <field name="name">Scheduled Call</field>
+            <field name="icon">fa-phone</field>
+            <field name="category">scheduled_phonecall</field>
+            <field name="delay_count">2</field>
+            <field name="sequence">7</field>
```

Comment:
Other mail.activity.types usually don't specify those fields, any reason to do it here?
I don't see anything in the spec.

Reply:
Those fields are specified on the mail_activity_data_call so, considering their similar nature, I specified it as well on the scheduled phonecall. I'll double check tomorrow to see if it is really needed or not.